packages/ps/test/commands/wait.test.ts

```diff
@@ -0,0 +1,122 @@
+import {expect, test} from '@oclif/test'
+
+const API_HOST = 'https://api.heroku.com'
+const APP_NAME = 'wubalubadubdub'
+
+const CURRENT = {
+  id: '00000000-0000-0000-0000-000000000002',
+  version: 23
+}
+
+const PREVIOUS = {
+  id: '00000000-0000-0000-0000-000000000001',
+  version: 22
+}
+
+const withRelease = test
+  .nock(API_HOST, api => api
+    .get(`/apps/${APP_NAME}/releases`)
+    .reply(200, [CURRENT])
+  )
+
+describe('heroku ps:wait', () => {
+  test
+    .nock(API_HOST, api => api
+      .get(`/apps/${APP_NAME}/releases`)
+      .reply(200, [])
+  )
+    .stderr()
+    .command(['ps:wait', '--app', APP_NAME])
+    .it('warns and exits 0 if no releases', ctx => {
+      expect(ctx.stderr).to.equal(` ›   Warning: App ${APP_NAME} has no releases\n`)
+    })
+
+  withRelease
+    .nock(API_HOST, api => api
+      .get(`/apps/${APP_NAME}/dynos`)
+      .reply(200, [
+        {release: CURRENT, state: 'up', type: 'web'}
+      ])
+  )
+    .command(['ps:wait', '--app', APP_NAME])
+    .it('exits with no output if app is already on the latest release')
```

Comment:
Should I be asserting no output explicitly or is that done automagically when `stderr()` is not called?